infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/plugins/CorfuReplicationClusterManagerBaseAdapter.java

```diff
@@ -39,7 +39,7 @@ public void setLocalEndpoint(String endpoint) {
      * @param newTopologyConfigMsg
      */
     public synchronized void updateTopologyConfig(TopologyConfigurationMsg newTopologyConfigMsg) {
-        if (newTopologyConfigMsg.getTopologyConfigID() > topologyConfig.getTopologyConfigID()) {
+        if (newTopologyConfigMsg.getTopologyConfigID() >= topologyConfig.getTopologyConfigID()) {
```

Comment:
Can we add a note in here saying that `>=` is fine here since it's only being utilized in tests.